pango/src/enums.rs

```diff
@@ -0,0 +1,147 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[cfg(any(feature = "v1_50", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+use glib::{
+    error::ErrorDomain,
+    translate::*,
+    value::{FromValue, ToValue},
+    Quark, StaticType,
+};
+#[cfg(any(feature = "v1_50", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+use std::fmt;
+
+#[cfg(any(feature = "v1_50", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+#[derive(Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Clone, Copy)]
+#[non_exhaustive]
+#[doc(alias = "PangoLayoutDeserializeError")]
+pub enum LayoutDeserializeError {
```

Comment:
Why make this manual?

Reply:
See the commit & todo in the code. gir uses a wrong error quark function name (that doesn't even exists??) and we can't disable only that part of the enum

Reply:
Isn't there a configuration for the error quark?

Reply:
I looked the config code but didn't find any, ideally the code in library postprocessing needs to be fixed as the quark is properly defined on the gir file.